src/devTools.js

```diff
@@ -112,13 +116,18 @@ export function connect(options = {}) {
     },
     send: (action, payload) => {
```

Comment:
Can we make this `send` an async function and `await hostnamePromise` before passing it into `send()`?

Reply:
done